project-keeper/src/main/java/com/exasol/projectkeeper/validators/release/ReleaseValidationBuilder.java

```diff
@@ -0,0 +1,49 @@
+package com.exasol.projectkeeper.validators.release;
+
+import java.nio.file.Path;
+import java.util.List;
+
+import com.exasol.projectkeeper.Validator;
+import com.exasol.projectkeeper.validators.changesfile.ChangesFile;
+import com.exasol.projectkeeper.validators.changesfile.ChangesFileIO;
+
+/**
+ * This class creates {@link Validator}s that check if the project satisfies all preconditions for a release.
+ */
+public class ReleaseValidationBuilder {
```

Comment:
```suggestion
public class ReleaseValidatorsBuilder {
```

Reply:
Or how about simply `ReleaseValidators`? Maybe the method `validators()` could then be renamed to `get()`? Or `build()?`

Reply:
Looking at the usage in `ProjectKeeper.java` it could be a static method for now, but maybe you know about future additions?
`ReleaseInspector` could also be a good name, and then `ReleaseInspector.validators()` sounds fine to me.

Reply:
I renamed the class, but I would like to not use a static method because it gets to many parameters.